### PR TITLE
Patch for Spanish translation.

### PR DESCRIPTION
**Pull request by joser93**
Greetings.

I modiffied a bit from the https://github.com/linkwarden/linkwarden/pull/714 because I noticed some fixes to be made, thanks fo @phampyk.

### CODE REVIEW

**Comment by phampyk** [comment]
Thanks for that, I should proofread my stuff better, I missed a whole sentence. Also bad habit of using "email" even when speaking Spanish.

**Comment by joser93** [comment]
> Thanks for that, I should proofread my stuff better, I missed a whole sentence. Also bad habit of using "email" even when speaking Spanish.

I get it lol. also happens to me and happens more often that you think :smile:

File: public/locales/es/common.json [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+  "create_new_user": "Crear nuevo usuario",
+  "placeholder_johnny": "Johnny",
+  "placeholder_email": "johnny@ejemplo.com",
+  "placeholder_john": "john",
+  "user_created": "¡Usuario creado!",
+  "fill_all_fields_error": "Por favor, rellena todos los campos.",
+  "password_change_note": "<0>Nota:</0> Por favor, asegúrate de informar al usuario que necesita cambiar su contraseña.",
+  "create_user": "Crear usuario",
+  "creating_token": "Creando token...",
+  "token_created": "¡Token creado!",
```
Comment (jmiguelr) [inline public/locales/es/common.json:273]: I'd remove the exclamation marks, I presume it's not needed to have so much emphasis // Creo que las exclamaciones no son necesarias en este contexto (pero es solo una opionion)

File: public/locales/es/common.json [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+  "access_token_created": "Token de acceso creado",
+  "token_creation_notice": "Tu nuevo token ha sido creado. Por favor, cópialo y guárdalo en un lugar seguro. No lo mostraremos de nuevo.",
+  "copied_to_clipboard": "¡Copiado al portapapeles!",
```
Comment (jmiguelr) [inline public/locales/es/common.json:276]: Same as above, no need to emphasize IMHO // Lo mismo de arriba, en mi opinion sobran las exclamaciones

**Review by jmiguelr — COMMENTED** [review]
Estaba planeando hacer la traducción cuando he visto que ya estaba en marcha. Gracias por el trabajo, a ver si podemos tener esta pedazo de aplicación en español cuanto antes. 

Las sugerencias que he puesto son, por supuesto, apreciaciones personales, asi que tómalas como tales. Si no te gustan, ignóralas sin problema :-)

**Comment by jmiguelr** [comment]
Por cierto, no se si sería conveniente (ni si es posible) usar como locale  es-ES para diferenciar de posibles traducciones latinoamericanas en el futuro

//

By the way: maybe it could be convenient to use es-ES as locale so we have now a Spanish-Spain translation but opened to future latin america translations.

File: public/locales/es/common.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (joser93) [reply public/locales/es/common.json:273]: As far that I know, this message is a toast or notification, so, exclamation marks are valid in the context that is being used. Beside that, the translation should keep the format from the english translation, if on a certain point the english translation changes, all of them should follow that.

**Comment by daniel31x13** [comment]
Hey, big thanks for your contributions.

I added/removed some fields from the main `en` locale file, could you please make another PR fixing them? For now I'll be merging this one.

**Pull request merged**